requests/serializers.py

```diff
@@ -58,14 +59,20 @@ def get_reimbursement(self, obj):
         try:
             volunteer = Volunteer.objects.get(request=obj)
             reimbursement = Reimbursement.objects.get(volunteer=volunteer)
-            return { "reimbursement_supporter":reimbursement.volunteer.supporter.display_name, 
-            "reimbursement_supporter_id": reimbursement.volunteer.supporter.id, 
-            "reimbursement_status": reimbursement.status, 
-            "reimbursement_amount": reimbursement.amount,
-            "reimbursement_id": reimbursement.id}
+            return ReimbursementSerializer(reimbursement).data
         except:
             return None
-    
+
+    def get_delivery_status(self, obj):
+        try:
+            volunteer = Volunteer.objects.get(request=obj)
+            if volunteer.status == 'Delivered':
```

Comment:
Sometimes in big projects I have found it nice to store strings like this in constants so we can reference a variable name everywhere. That way, if the string needs to change, it only needs to change in one place.

So maybe in `requests/request_form_choices.py` define:

```
VOLUNTEERING_STATUS_SIGNED_UP = 'Signed Up'
VOLUNTEERING_STATUS_READY = 'Ready'
VOLUNTEERING_STATUS_DELIVERED = 'Delivered'

VOLUNTEERING_STATUS_CHOICES = (
    (VOLUNTEERING_STATUS_SIGNED_UP, 'Signed Up'),
    (VOLUNTEERING_STATUS_READY, 'Ready'),
    (VOLUNTEERING_STATUS_DELIVERED, 'Delivered')
)
```

and then the logic in here can reference the constant like this? This means everywhere you reference any `volunteer.status` you can use the constants! What do you think about this kind of pattern?

```
return volunteer.status if volunteer.status == VOLUNTEER_STATUS_DELIVERED else 'On the way'
```

Reply:
+1 for constants!

Reply:
This is so great. I made the change! 
Thanks a lot bri :pray: